assets/stylesheets/components/_typography.scss

```diff
@@ -156,6 +156,35 @@
   }
 }
 
+.component-header-box {
+  border: 3px solid;
+  display: inline-block;
+  font-family: $condensed-extra-font-face;
+  font-size: $font-size-h1;
+  font-weight: bold;
+  letter-spacing: 4px;
+  padding: 25px;
+  text-align: center;
+  text-transform: uppercase;
+
+  &.text-orange {
```

Comment:
Could you set `border-color: currentColor` in the main `.component-header-box` instead of having `text-*` specific ones so we do not have to add them each time. (I could be missing something please let me know if I am)